.github/CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 # Order is important. The last matching pattern has the most precedence.
 
 *             @chef/inspec-maintainers
-.expeditor/** @chef/engineering-services
-omnibus/**    @chef/engineering-services
+.expeditor/** @chef/jex-team
+omnibus/**    @chef/jex-team
```

Comment:
Should this be JEX or BEN?

Reply:
Yes. I updated. I'll also try and loop back around to other Omnibus based projects.